ddtrace/internal/debug.py

```diff
@@ -137,3 +138,91 @@ def collect(tracer):
         partial_flush_enabled=tracer._partial_flush_enabled,
         partial_flush_min_spans=tracer._partial_flush_min_spans,
     )
+
+
+def pretty_collect(tracer, color=True):
+    class bcolors:
+        HEADER = "\033[95m"
+        OKBLUE = "\033[94m"
+        OKCYAN = "\033[96m"
+        OKGREEN = "\033[92m"
+        WARNING = "\033[93m"
+        FAIL = "\033[91m"
+        ENDC = "\033[0m"
+        BOLD = "\033[1m"
+
+    info = collect(tracer)
+
+    info_pretty = """{0}{1}Tracer Configurations:{2}
+    Tracer enabled: {3}
+    Debug logging: {4}
+    Writing traces to: {5}
+    Agent error: {6}
+    App Analytics enabled(deprecated): {7}
+    Log injection enabled: {8}
+    Health metrics enabled: {9}
+    Priority sampling enabled: {10}
+    Partial flushing enabled: {11}
+    Partial flush minimum number of spans: {12}
+    {13}{14}Tagging:{15}
+    DD Service: {16}
+    DD Env: {17}
+    DD Version: {18}
+    Global Tags: {19}
+    Tracer Tags: {20}""".format(
+        bcolors.OKBLUE,
+        bcolors.BOLD,
+        bcolors.ENDC,
+        info.get("tracer_enabled"),
+        info.get("debug"),
+        info.get("agent_url") or "Not writing at the moment, is your tracer running?",
+        info.get("agent_error") or "None",
+        info.get("analytics_enabled"),
+        info.get("log_injection_enabled"),
+        info.get("health_metrics_enabled"),
+        info.get("priority_sampling_enabled"),
+        info.get("partial_flush_enabled"),
+        info.get("partial_flush_min_spans") or "Not set",
+        bcolors.OKGREEN,
+        bcolors.BOLD,
+        bcolors.ENDC,
+        info.get("service") or "None",
+        info.get("env") or "None",
+        info.get("dd_version") or "None",
+        info.get("global_tags") or "None",
+        info.get("tracer_tags") or "None",
```

Comment:
```suggestion
        info.get("tracer_tags") or "None",
        blue=bcolors.OKBLUE,
```

Just did blue as an example here